code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -10,13 +10,14 @@ import StageDetails from './stage_details.jsx';
  */
 var CourseProgress = React.createClass({
   propTypes: {
+    currentLevelId: React.PropTypes.string,
     display: React.PropTypes.oneOf(['dots', 'list']).isRequired,
     stages: React.PropTypes.arrayOf(STAGE_TYPE)
   },
 
   getRow(stage) {
     if (this.props.display === 'dots') {
-      return <CourseProgressRow stage={stage} key={stage.name} />;
+      return <CourseProgressRow stage={stage} key={stage.name} currentLevelId={this.props.currentLevelId} />;
```

Comment:
@breville This is how you pass props to child components (i.e. The CourseProgressRow component will now have a prop named `currentLevelId` set to the same value as this component's `currentLevelId`